src/modules/email/email.service.ts

```diff
@@ -0,0 +1,29 @@
+import Config from '../../config';
+import * as sgMail from '@sendgrid/mail';
+import {SendData} from './interfaces/email.interface'
+import { Injectable } from '@nestjs/common';
+
+
+const defaults = {
+    from: 'Coding Coach <no-reply@mail.codingcoach.io>',
```

Comment:
Should we put this in the `.env` file so we can update it easily? That way we won't update the code to set this value

Reply:
What is a scenario in which this value would change?

Reply:
maybe we want to use `codingcoach.io` instead of `mail.codingcoach.io`, maybe we would want to change `Coding Coach` to `Emma from Coding Coach`. I don't know yet, but I would like to keep it flexible.

Reply:
@crysfel discussed this in Slack and determined that this value shouldn't change often enough to warrant an environment-specific setting.  However, it would still benefit from being moved to the config.  So I am going to do that.

Reply:
This has been updated.